src/dlc2kinematics/utils/auxiliaryfunctions.py

```diff
@@ -228,17 +228,16 @@ def doubleangle_calc(pos):
     pos = pos.reshape((3, 3))
     # print(pos)
     u = pos[1, :] - pos[0, :]
-    v = pos[2, :] - pos[1, :]
+    v = pos[1, :] - pos[2, :]
 
     rel = v - u
 
     x = rel[0]
     y = rel[1]
     z = rel[2]
-
+    # yaw, pitch relative to the vertical plane (x-z plane)
     yaw = np.arctan(x / z) * 180 / np.pi
```

Comment:
We can probably improve that line here, using atan2, to handle all quadrants and zero denominators:
```suggestion
    yaw = np.rad2deg(np.arctan2(x, z))
```